galgebra/ga.py

```diff
@@ -529,7 +534,7 @@ def mv(self, root=None, *args, **kwargs):
         Instanciate and return a multivector for this, 'self',
         geometric algebra.
         """
-        (self.mv_I, self.mv_basis, self.mv_x) = mv.Mv.setup(ga=self)
+        (_, self.mv_basis, self.mv_x) = mv.Mv.setup(ga=self)
```

Comment:
Every time you construct a new vector, `self.mv_x` changes:
```
In [26]: g.mv('A', 'vector'); ax = g.mv_x; ax is g.mv_x
Out[26]: True

In [27]: g.mv('B', 'vector'); bx = g.mv_x; bx is g.mv_x
Out[27]: True

In [28]: ax is bx
Out[28]: False
```

At least after this patch, `mv_I` does not.

This whole line needs to go, but that can come in a different patch

Reply:
Don't like the inconsistency here, can we do something for them in one go? So the reviewer can see the bigger picture and final outcome.

Reply:
Perhaps I'll swap the order of this patch and the upcoming one

Reply:
#79

Reply:
So is merging this blocked by the currently failing #79 ?

I'm ready to merge this PR if the order doesn't matter. Since other issues are well answered and this issue is expected to be resolved iin #79 .

Reply:
Let's put 79 in first, they'll conflict.

Can I get your opinion on the failure in #79?